src/protagonist/API.Tests/Features/Queues/Validation/QueuePostValidatorTests.cs

```diff
@@ -11,11 +11,15 @@ namespace API.Tests.Features.Queues.Validation;
 public class QueuePostValidatorTests
 {
     private readonly QueuePostValidator sut;
-
+    private readonly QueuePostValidator sutWithOldDcEmulation;
+    
     public QueuePostValidatorTests()
     {
-        var apiSettings = new ApiSettings { MaxBatchSize = 4 };
-        sut = new QueuePostValidator(Options.Create(apiSettings));
+        var apiSettingsA = new ApiSettings { MaxBatchSize = 4 };
+        sut = new QueuePostValidator(Options.Create(apiSettingsA));
+        
+        var apiSettingsB = new ApiSettings { EmulateOldDeliveryChannelProperties = true};
+        sutWithOldDcEmulation = new QueuePostValidator(Options.Create(apiSettingsB));

```

Comment:
Where there are changes in SUT other tests use a `GetSut()` method that takes parameters to control how it is configured.